### DISCUSSION TITLE
Syntax for the IS IN filter

### DISCUSSION THREAD

**Discussion by NicolasFabre81** [Budibase/budibase#8912] [Help] [answered]
I want to filter my data provider with the "is in" and several values...
What is the correct syntax please? I tried almost everything, nothing works...
<img width="585" alt="image" src="https://user-images.githubusercontent.com/28592069/205335528-04ffa909-5ae5-475f-a470-7497dd8df167.png">

**Comment by melohagan** [comment]
Hey @NicolasFabre81 

What data type is perimetre?

**Reply by NicolasFabre81** [reply]
Hello @melohagan  , it is simple text (varchar in the postgresql DB)

**Comment by micebrain** [comment, marked as answer]
Hi @NicolasFabre81 
I found out that it is actually expecting an array. Can you try with the Handlebar helper like this:
```
{{ split 'France,Spain' }} 
```

**Reply by NicolasFabre81** [reply]
Thanks four your reply, but does not work...
I also tried {{ arrayify 'France,Spain' }} ...

**Reply by NicolasFabre81** [reply]
With one value, this works : IS IN {{arrayify 'France'}}

**Reply by melohagan** [reply]
It works if you use JavaScript to return an array, e.g. `return ["Snowball", "Fluffy"]`

This is a general problem with Handlebars as they always return a string.

**Reply by NicolasFabre81** [reply]
I just discovered it at the moment ! Thanks
